sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -231,11 +231,15 @@ def sample(
         self.potential_ = self._prepare_potential(method)  # type: ignore
 
         init_fn = self._build_mcmc_init_fn(
-            self.proposal, self.potential_fn, transform=self.theta_transform
+            self.proposal,
+            self.potential_fn,
+            transform=self.theta_transform,
+            init_strategy=init_strategy,  # type: ignore
         )
         initial_params = torch.cat(
             [init_fn() for _ in range(num_chains)]  # type: ignore
         )
+        print("initial_params", initial_params)
```

Comment:
is this intended? if yes, add more details: `f"initial MCMC parameters: {initial_params}"`

Reply:
ups, no. Thanks!